src/main/scala/org/codeoverflow/chatoverflow/requirement/service/rcon/impl/RconInputImpl.scala

```diff
@@ -0,0 +1,19 @@
+package org.codeoverflow.chatoverflow.requirement.service.rcon.impl
+
+import org.codeoverflow.chatoverflow.WithLogger
+import org.codeoverflow.chatoverflow.api.io.input.RconInput
+import org.codeoverflow.chatoverflow.registry.Impl
+import org.codeoverflow.chatoverflow.requirement.InputImpl
+import org.codeoverflow.chatoverflow.requirement.service.rcon.RconConnector
+
+@Impl(impl = classOf[RconInput], connector = classOf[RconConnector])
+class RconInputImpl extends InputImpl[RconConnector] with RconInput with WithLogger {
+  override def getCommandOutput(command: String): String = sourceConnector.get.sendCommand(command)
+
+  /**
+    * Start the input, called after source connector did init
+    *
+    * @return true if starting the input was successful, false if some problems occurred
+    */
+  override def start(): Boolean = sourceConnector.get.isLoggedIn
```

Comment:
As pointed out in my other comment the check if logged in should be in the start method of the connector.  
Here you can just return true as no further stuff has to be done.

The same can be said for the OutputImpl